extensions/wikia/FlowTracking/scripts/initCreatePageTracking.js

```diff
@@ -6,11 +6,13 @@ require([
 	'jquery',
 	'wikia.window'
 ], function (flowTracking, flowTrackingCreatePage, QueryString, mw, $, window) {
+	var redLinkFlow = mw.config.get('wgNamespaceNumber') === -1 ?
+			window.wgFlowTrackingFlows.CREATE_PAGE_SPECIAL_REDLINK :
+			window.wgFlowTrackingFlows.CREATE_PAGE_ARTICLE_REDLINK,
+		createButtonFlow = window.wgFlowTrackingFlows.CREATE_PAGE_CREATE_BUTTON;
```

Comment:
Hmm, it's either leaning towards keeping flows as local vars (current approach) or using this static value in https://github.com/Wikia/app/pull/11589/files#diff-2e6e1784157fc2addcc6e0435179abedR36

I have no strong preference either way, both are readable